pkg/convert/go.go

```diff
@@ -0,0 +1,69 @@
+package convert
```

Comment:
strange filename go.go

Reply:
We were using the same file name in POC.

Reply:
We had the following transformations for different use cases:

- transform from/to GRPC messages (package transform)
- transform from/to files (package convert)

I would prefer to have a separate package for each one. Package name suggestions are welcome.